release/release.go

```diff
@@ -853,3 +913,17 @@ As always, we welcome and appreciate feedback from our community of users. Pleas
 - [Check out our documentation](https://rancher.com/docs/k3s/latest/en/) for guidance on how to get started or to dive deep into K3s.
 - [Read how you can contribute here](https://github.com/rancher/k3s/blob/master/CONTRIBUTING.md)
 {{ end }}`
+
+const uiReleaseNoteTemplate = `
```

Comment:
I am not sure about this, but shouldn't these constants for templates be defined at the beginning of the file?